week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
```

빈 태그에 height 값을 줌으로써 위 여백을 설정하는 것보다는,
body 태그에서 `margin-top` 또는 `padding-top`을 통해 위 여백을 설정할 수 있습니다.
```suggestion
html,body{
    padding-top: 90px;
    background-color: #c1bcea;
}
```